custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -647,11 +647,16 @@ def check_prediction_side_effects(self, target_temp_location=None):
             )
 
             if self.target_type == TargetType.TRANSFORM:
```

Comment:
note to self: be sure to check case where both input and output of transform are sparse